jpa-criteria/impl/src/main/java/com/blazebit/persistence/criteria/impl/InternalQuery.java

```diff
@@ -528,6 +531,62 @@ private SubqueryBuilder<?> renderSubqueryFrom(SubqueryInitiator<?> initiator, Re
         return cb;
     }
 
+    public LeafOngoingSetOperationSubqueryBuilder<?> renderSubqueryFrom(LeafOngoingSetOperationSubqueryBuilder<?> initiator, RenderContextImpl context) {
```

Comment:
Rendering between subqueries and outer queries is slightly different here, as there is no `LeafOngoingSetOperationSubqueryInitiator` and therefore no abstraction shared with `SubqueryInitiator` (which was intentionally separated from `BaseSubqueryBuilder`, which is the abstraction you most likely might want to use here instead of `LeafOngoingSetOperationSubqueryBuilder`).

Reply:
Sounds good to me.